test/util/cart_utils.py

```diff
@@ -301,3 +311,27 @@ def print(self, cmd):
 
         self.stdout.info(cmd)
         self.progress_log.info(cmd)
+
+    def log_check(self, cartobj):
+	"""Check log files for consistency """
+
+	logparse = cartobj.params.get("logparse", "/run/tests/*/")
+        if logparse is None:
+		return
+
+        """Check log files for consistency """
+        strict_test = False
+	print ("Parsing log path", cartobj.log_path)
+	if not os.path.exists(cartobj.log_path):
+		print ("Path does not exist")
+		return
+
+	for filename in os.listdir(cartobj.log_path):
+	    log_file = os.path.join(cartobj.log_path, filename)
+	    if os.path.isfile(log_file):
+	        print ("Parsing ", log_file)
+                cl = cart_logparse.LogIter(log_file)
+                c_log_test = cart_logtest.LogTest(cl)
+	        c_log_test.check_log_file(strict_test)
+	    else:
```

Comment:
style - but I'd have done "if not os.path.isfile(log_file): print...; continue" and then called logparse with one less levels of indentation.

Reply:
okay